### DISCUSSION TITLE
Checking individual microinverters through the ECU?

### DISCUSSION THREAD

**Discussion by cralwalker** [ksheumaker/homeassistant-apsystems_ecur#172] [Q&A]
I have 500W AP Systems branded microinverters by Greenstar and wonder how I can monitor them.
The ECU is version 3, for the YC500.

![image](https://user-images.githubusercontent.com/14145686/230016718-a18eb6a0-0a70-4073-bc52-99794eb3be44.png)

**Comment by HAEdwin** [comment]
The ECU-3 can only be installed using ethernet and web gui. Therefor it is not compatible with this integration unfortunately.
